kernel/src/main/java/org/kframework/kompile/Kompile.java

```diff
@@ -136,12 +136,14 @@ public CompiledDefinition run(File definitionFile, String mainModuleName, String
             }
         }
         Definition parsedDef = parseDefinition(definitionFile, mainModuleName, mainProgramsModuleName, excludedModuleTags);
-        sw.printIntermediate("Parse definition [" + definitionParsing.parsedBubbles.get() + "/" + (definitionParsing.parsedBubbles.get() + definitionParsing.cachedBubbles.get()) + " rules]");
+        sw.printIntermediate("Parse rules [" + definitionParsing.parsedBubbles.get() + "/" + (definitionParsing.parsedBubbles.get() + definitionParsing.cachedBubbles.get()) + " rules]");
 
         files.saveToKompiled("parsed.txt", parsedDef.toString());
         checkDefinition(parsedDef, excludedModuleTags);
         sw.printIntermediate("Run definition checks");
 
+        sw.printIntermediate("Validate definition");
+
```

Comment:
This is redundant with 2 lines above (which I added in a previous PR) which says "Run definition checks.". So either remove this or just update the text of the one above.